packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -19,6 +19,11 @@ export interface ISummaryStats {
     totalBlobSize: number;
 }
 
+export interface ITopLevelSummaryStats extends ISummaryStats{
+    dataStoreCount: number;
+    summarizedDataStoreCount: number;
```

Comment:
Aren't we counting summarizedUnreferencedDataStoreCount now?

Reply:
not now... i think Navin is doing that in #6553 , then when our PRs are both merged, there can be a third PR to log this.

Reply:
Let's add the unreferenced count in a separate PR once my PR goes in. I also think summarizedUnreferencedDataStoreCount does not add any value. Just unreferenced data store count should be good enough.